tracer/src/Datadog.Trace.Tools.Runner/Utils.cs

```diff
@@ -509,9 +534,162 @@ internal static bool IsAlpine()
                 envVars["COR_PROFILER_PATH_ARM64"] = tracerProfilerArm64;
             }
 
+            if (!string.IsNullOrEmpty(devPath))
+            {
+                envVars["DEVPATH"] = devPath;
+            }
+
             return envVars;
         }
 
+        private static bool EnsureDatadogTraceIsInTheGac(string tracerHome)
+        {
+            // We try to ensure Datadog.Trace.dll is installed in the gac for compatibility with .NET Framework fusion class loader
+            // Let's find gacutil, because CI Visibility runs with the SDK / CI environments it's probable that's available.
+            Log.Debug("EnsureDatadogTraceIsInTheGac: Looking for gacutil");
+            var cmdResponse = ProcessHelpers.RunCommand(new ProcessHelpers.Command("where", "gacutil"));
+            if (cmdResponse?.ExitCode == 0 &&
+                cmdResponse.Output.Split(["\n", "\r\n"], StringSplitOptions.RemoveEmptyEntries) is { Length: > 0 } outputLines &&
+                outputLines[0] is { Length: > 0 } gacPath)
+            {
+                Log.Debug("EnsureDatadogTraceIsInTheGac: gacutil was found.");
+                var cmdGacListResponse = ProcessHelpers.RunCommand(new ProcessHelpers.Command(gacPath, "/l"));
+                if (cmdGacListResponse?.ExitCode == 0)
+                {
+                    if (cmdGacListResponse.Output?.Contains(typeof(TracerConstants).Assembly.FullName!, StringComparison.OrdinalIgnoreCase) != true)
+                    {
+                        // Datadog.Trace is not in the GAC, let's try to install it.
+                        // We run the gacutil /i command using runas verb to elevate privileges
+                        Log.Warning("EnsureDatadogTraceIsInTheGac: Datadog.Trace is not in the GAC, let's try to install it.");
+                        if (FileExistsOrNull(Path.Combine(tracerHome, "net461", "Datadog.Trace.dll")) is { } datadogTraceDllPath &&
+                            ProcessHelpers.RunCommand(new ProcessHelpers.Command(gacPath, $"/if {datadogTraceDllPath}", verb: "runas")) is { } cmdGacInstallResponse)
```

Comment:
I believe this may trigger an UAC prompt? You may want to display something in the console to indicate to the user what you're doing

Reply:
True, done in https://github.com/DataDog/dd-trace-dotnet/pull/5129/commits/2d37a880174fb7fcb774d5ff7d8a79be24b1ebd1

Looks like this: 
![image](https://github.com/DataDog/dd-trace-dotnet/assets/69803/f6cc5b8f-6ea5-4a49-a4b6-386f4cb2d437)